packages/hydrogen-react/src/load-script.tsx

```diff
@@ -32,6 +38,13 @@ export function loadScript(
     } else {
       document.body.appendChild(script);
     }
+
+    const attributes = options?.attributes;
+    if (attributes) {
+      Object.keys(attributes).forEach((key) => {
+        attributes[key] && script.setAttribute(key, attributes[key]);
```

Comment:
Is the `attributes[key] &&` there to not override attributes like `src`?

Reply:
mmm turns out I need to take it off for

```
useLoadScript({
  attribute: {
    test: '',
  }
})
```

This generates `<script test="" />` which is valid